pkg/agent/multicast/mcast_route_test.go

```diff
@@ -66,6 +66,46 @@ func TestParseIGMPMsg(t *testing.T) {
 	}
 }
 
+func TestDeleteInboundMrouteEntryByGroup(t *testing.T) {
+	mRoute := newMockMulticastRouteClient(t)
+	err := mRoute.initialize(t)
+	assert.Nil(t, err)
```

Comment:
Use require.NoError to avoid unexpected failure in the following logic.

Reply:
upated